src/css/custom.css

```diff
@@ -156,3 +158,32 @@ html[data-theme="dark"] {
   padding: 1rem;
   border-left: 3px solid var(--ifm-tabs-color-active-border);
 }
+
+/* Homepage specific styling */
```

Comment:
I think we can also hide the "Getting Started" title on the landing page to save vertical space.